### PR TITLE
additional logging option: CLI

### PR DESCRIPTION
**Pull request by enbrnz**
on Windows you need to open a powershell
on Linux just a standard terminal

Just a quick addition to the docs. Wording can probably be improved

### CODE REVIEW

**Review by michaelstingl — APPROVED** [review]
LGTM, but I’d prefer to wait for @TheOneRing ‘s approval

File: docs/modules/ROOT/pages/troubleshooting.adoc [hunk 1]
```diff
@@ -118,6 +118,21 @@ owncloud --logdir /tmp/owncloud_logs --logexpire 48
...
 Adding the `--logdebug` flag increases the verbosity of the generated log files.
 
+==== Logging in the Console
+
+If the ownCloud client isn't able to start and immediately crashes the first two options are not available. 
+Therefore it might need to be necessary to start the ownCloud client using the command line in order to be see the error message
+
+On Linux and Mac simply open the terminal and run `owncloud`.
+
+On Windows open a PowerShell and run the following command:
```
Comment (TheOneRing) [inline docs/modules/ROOT/pages/troubleshooting.adoc:128]: --logfile - --logflush is also recommended on unix we will just get a more detailed log.

--logdebug will create a more detailed log.

The pipe on Windows is required as usually no console is attached to a gui process so the output is lost.

Comment (enbrnz) [reply docs/modules/ROOT/pages/troubleshooting.adoc:131]: Thanks for your input!
Better like that?

**Comment by TheOneRing** [comment]
I rebased the change to 2.7 so we will get it in 2.7 and master

**Review by TheOneRing — APPROVED** [review]

**Pull request merged**